src/authorizations/components/redesigned/TokenRow.tsx

```diff
@@ -110,7 +111,16 @@ class TokensRow extends PureComponent<Props> {
   }
 
   private handleClone = () => {
-    this.props.onClone(this.props.auth.id)
+    const {description} = this.props.auth
+
+    const alltokenNames = Object.values(this.props.authorizations).map(
```

Comment:
for consistency, let's call this `allTokenDescriptions` since on the `auth` object it's called `description` - that seems to be the canonical name. The call to `incrementCloneName(alltokenNames, description)` is slightly confusing because it's looking through the list of token _names_ but it's adding a _description_. A new engineer who's not as familiar with our tokens and auths might be very confused here.